### PR TITLE
BETA: Register servers.is-a.dev

### PR DESCRIPTION
**Pull request by Rage65**
Added `servers.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/servers.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Rage65",
+        "email": "123thetechguy@gmail.com"
+    },
+    "record": {
+        "CNAME": "rage65.duckdns.org"
```
Comment (CuteDog5695) [inline domains/servers.json:7]: site seems to not be accessible

File: domains/servers.json
Comment (CuteDog5695) [inline domains/servers.json:1]: will not be released since it has a high risk of being abused

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/servers.json (hunk 1 above)
Comment (Rage65) [reply domains/servers.json:7]: Sorry that is my tailscale adress for my server

File: domains/servers.json
Comment (Rage65) [reply domains/servers.json:1]: Ok 😔

**Pull request closed without merging**